spec/index.html

```diff
@@ -358,17 +358,17 @@ <h2>Abstract</h2>
     <section id="introduction">
       <h2>Introduction</h2>
       <p>The growing number of SPARQL query services offer data consumers an opportunity to merge data distributed across the Web. This specification defines the syntax and semantics of the
-      <code>SERVICE</code> extension to the SPARQL 1.1 Query Language. This extension allows a query author to direct a portion of a query to a particular SPARQL endpoint. Results are returned to the
+      <code>SERVICE</code> extension to the SPARQL Query Language. This extension allows a query author to direct a portion of a query to a particular SPARQL endpoint. Results are returned to the
       federated query processor and are combined with results from the rest of the query.</p>
       <section id="docConventions">
         <h3>Document Conventions</h3>
         <section id="docNamespaces">
           <h4>Namespaces</h4>
-          <p>This document uses the same namespaces as from the <a class="inform" href="http://www.w3.org/TR/2011/WD-sparql11-query-20110512/#docNamespaces">SPARQL 1.1 Query document</a>.</p>
+          <p>This document uses the same namespaces as from the [[[SPARQL12-QUERY]]].</p>
```

Comment:
Without the trailing "document", I think we also should to drop the preceding "the" so that it renders as "as from SPARQL 1.2 Query Language." Alternatively, keep "document" (or something similar).